tests/schedulers/test_scheduler_dpm_multi.py

```diff
@@ -201,7 +201,7 @@ def test_full_loop_no_noise_thres(self):
         sample = self.full_loop(thresholding=True, dynamic_thresholding_ratio=0.87, sample_max_value=0.5)
         result_mean = torch.mean(torch.abs(sample))
 
-        assert abs(result_mean.item() - 0.6405) < 1e-3
+        assert abs(result_mean.item() - 1.1364) < 1e-3
```

Comment:
`1.1364` seems like a magic number to me.

Reply:
I think basically all our tests have magic numbers haha. It's just a sanity check for consistency. In this case, the sample_max_value is no longer being used as the min clamp so that's why the output changed :)